Project/frontend/Dockerfile.frontend

```diff
@@ -0,0 +1,15 @@
+FROM node:16-alpine 
+
+WORKDIR /app
+
+ENV PATH /app/node_modules/.bin:$PATH
+
+COPY package.json ./
+COPY package-lock.json ./
+
+RUN npm install --silent
+RUN npm install
+
+COPY . ./
```

Comment:
To jest bardzo, ale to bardzo zła praktyka. Powinno kopiować się tylko i wyłącznie to co jest niezbędne. Nie mamy w projekcie .dockerignore, więc wrzucisz tu dosłownie wszystko - razem z node-modules które dopiero zainstalowałeś oraz plikiem .gitignore.

Reply:
oke